src/tools/pontoon/ui.md

```diff
@@ -4,115 +4,123 @@
 
 In the bottom left corner, the FTL button (highlighted in yellow) allows to switch between the standard UI (*Simple FTL mode*) and the *Advanced FTL mode*, where it’s possible to edit the original syntax directly, as you would in a text editor. For details on the Fluent syntax, see [Fluent for localizers](../fluent/).
 
-![Translation workspace for Fluent string in source view](../../assets/images/pontoon/ui/workspace_ftl_sourceview.png)
+![Translation workspace for Fluent string in source view](../../assets/images/pontoon/ui/workspace_ftl_sourceview.png "Screenshot of the translation workspace for Fluent string in source view")
 
 Note that the FTL button’s text is green when in *Advanced FTL mode*.
 
-### Pontoon workspace resources
+### Translation tools and comments
 
-Built-in translation resources are located below the editing space. The three integrated resources are `HISTORY`, `MACHINERY`, and `LOCALES`.
+Built-in translation tools are located in the rightmost column. For more information about `MACHINERY`, and `LOCALES`, see the page dedicated to [Pontoon’s translation resources](resources.md).
 
-![Translation Resources](../../assets/images/pontoon/ui/resources.png)
+#### Terminogy
 
-Learn how to use Pontoon’s translation resources [in this document](resources.md).
+The `TERMS` tab shows the definition for a term, in case the source string includes matches with the built-in [terminology](glossary.md#terminology). The matching term is also highlighted in the source string.
 
-## Pontoon in-context editor
+![Translation workspace for string with match in terminology](../../assets/images/pontoon/ui/workspace_terminology.png "Screenshot of the translation workspace for string with match in terminology")
 
-Pontoon’s in-context editor displays the translatable strings as they appear on the web page, alongside the editing space. This means that translators can see where the source string appears and how the translation looks in real time. Pontoon’s in-context editor shows the strings in context on the right with the translation workbench on the left.
+#### Source string comments
 
-![in-context editor](../../assets/images/pontoon/ui/incontext_edit.png)
+The `COMMENTS` tab is used to display existing **source string comments**, or add new ones. Source string comments, unlike translation comments, are associated with the string: it’s possible to have a comment in this section even if the string doesn’t have any suggestion or translation yet.
 
-When using the in-context editor, the sidebar features both string navigation and the editing space. To view the editing space, double click on a string. To return to string navigation, click `BACK TO LIST` at the top-left of the sidebar.
+They’re designed for team members to have a conversation about the source string, for example to clarify its meaning, or to get more information from project managers.
```

Comment:
In the next iteration, when we'll restructure the docs, we should be consistent and cover Terminology and Source string comment in a similar fashion as Machinery and Locales.